src/web/mjs/connectors/Seven8te.mjs

```diff
@@ -0,0 +1,12 @@
+import MH from './templates/MH.mjs';
+
+export default class Seven8te extends MH {
+
+    constructor() {
+        super();
+        super.id = 'seven8te';
```

Comment:
IDs are allowed to start with numbers, its just the class name that must start with an uppercase letter